.github/workflows/_runner-determinator.yml

```diff
@@ -98,26 +107,28 @@ jobs:
               return branch.split("/")[0] in {"main", "nightly", "release", "landchecks"}
 
 
-          def get_workflow_type(issue: Issue, username: str) -> Tuple[str, str]:
-              formatted_username = (
-                  f"@{username}"  # Add the @ symbol to match the format in the issue body
-              )
-
+          def get_workflow_type(issue: Issue, username_check: Iterable[str]) -> Tuple[str, str]:
               try:
-                  user_list = issue.get_comments()[0].body.split()
+                  first_comment = issue.get_comments()[0].body.strip("\n\t ")
 
-                  if user_list[0] == "!":
+                  if first_comment[0] == "!":
                       MESSAGE = "LF Workflows are disabled for everyone. Using meta runners."
                       return WORKFLOW_LABEL_META, MESSAGE
-                  elif user_list[0] == "*":
+                  elif first_comment[0] == "*":
                       MESSAGE = "LF Workflows are enabled for everyone. Using LF runners."
                       return WORKFLOW_LABEL_LF, MESSAGE
-                  elif formatted_username in user_list:
-                      MESSAGE = f"LF Workflows are enabled for {username}. Using LF runners."
-                      return WORKFLOW_LABEL_LF, MESSAGE
                   else:
-                      MESSAGE = f"LF Workflows are disabled for {username}. Using meta runners."
-                      return WORKFLOW_LABEL_META, MESSAGE
+                      user_w_at = first_comment.split()
+                      user_set = {
+                          f"@{susr}" for susr in (usr.strip("\n\t ") for usr in first_comment.split())
```

Comment:
What's susr?

Reply:
susr == stipped_usr...

python sometimes is not very readable when you code when you do functional...